packages/jwt-middleware/test/operation.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const JwtMiddleware = require('../');
```

Comment:
These tests are great

Reply:
Can we add some positive test cases also?

Reply:
Yes, that was going to be my next step.  Needed to figure out how to generate tokens on the fly and  provide iss/aud info offline.